routes/api.php

```diff
@@ -27,6 +29,9 @@
     Route::resource('tag', TagController::class)->except(['create', 'edit']);
     Route::resource('sender-identity', SenderIdentityController::class)->except(['create', 'edit']);
     Route::resource('campaign', CampaignController::class)->except(['create', 'edit']);
+    Route::resource('roles', RoleController::class)->except(['create', 'edit']);
+    Route::put('user/update-user-roles', [PermissionController::class, 'updateUserRoles'])->name('user.updateUserRoles');
+    Route::post('user/update-role-permissions', [PermissionController::class, 'updateRolePermissions'])->name('user.updateRolePermissions');
```

Comment:
these two routes should be part of UserController and RoleController classes respectively.